caluma/caluma_form/structure.py

```diff
@@ -175,9 +176,13 @@ def _store_question(self, question):
                 continue
             # Evaluator is only used for extracting answer transforms & friends,
             # so is field-independent here
-            for dependency_slug in self._evaluator.extract_referenced_questions(
-                jexl_expr
-            ):
+            referenced_questions = set(
+                itertools.chain(
+                    self._evaluator.extract_referenced_questions(jexl_expr),
+                    self._evaluator.extract_referenced_mapby_questions(jexl_expr),
```

Comment:
Let's refactor this in the future: "extract_referenced_questions" should return _all_ referenced questions.